$NetBSD: patch-tests_runtests,v 1.2 2020/03/23 16:49:07 riastradh Exp $

- Let the environment specify SORT so we can set it from the makefile.

--- tests/runtests.orig	2020-02-22 07:25:00.000000000 +0000
+++ tests/runtests
@@ -3,11 +3,13 @@
 status=0
 
 # We need sort from coreutils for -V
+if [ -z "${SORT-}" ]; then
 case `uname -s` in
 	Linux*) SORT=sort ;;
 	Darwin*) SORT=gsort ;;
 	*) SORT=sort ;;
 esac
+fi
 
 test_conversion () {
 	test=$1
